pkg/compactor/blocks_cleaner.go

```diff
@@ -295,10 +315,25 @@ func (c *BlocksCleaner) checkRunError(runType string, err error) {
 	}
 }
 
-func (c *BlocksCleaner) runActiveUserCleanup(ctx context.Context, jobChan chan *cleanerJob) {
+func (c *BlocksCleaner) runEmitMetricsWorker(ctx context.Context, jobChan <-chan *cleanerJob) {
```

Comment:
Since this metric worker and `emitUserMetrics` function are only used when partitioning compactor is enabled, maybe we can mention partition in the function name to make it clear.
Like `emitPartitionMetricsWorker` and `emitUserParititionMetrics`